config/flags_node.go

```diff
@@ -170,6 +170,12 @@ var (
 		Usage: "Marks vendor (distributor) of the node for collecting statistics. " +
 			"3rd party vendors may use their own identifier here.",
 	}
+	// FlagUIEnable enables built-in web UI for node.
+	FlagIncludeFailedNodes = cli.BoolFlag{
```

Comment:
Don't introduce this flag, all filtering "configuration" - is passing Tequilapi parameters. I see you used it only method `mAPI.Proposals()` which is not used.

Reply:
Removed it.